optaplanner-core/src/main/java/org/optaplanner/core/config/solver/SolverConfig.java

```diff
@@ -244,6 +247,10 @@ public static SolverConfig createFromXmlReader(Reader reader, ClassLoader classL
     @XmlElement(name = "entityClass")
     protected List<Class<?>> entityClassList = null;
     protected DomainAccessType domainAccessType = null;
+    @XmlTransient
+    protected Map<String, MemberAccessor> memberAccessorMap = null;
+    @XmlTransient
+    protected Map<String, SolutionCloner> solutionClonerMap = null;
```

Comment:
If these are gizmo specific, their name should probably start with "gizmo" (so gizmoMemberAccessorMap) to avoid confusing for users of the programmatic API

Reply:
Not quite Gizmo specific; if they are present, the member accessor and solution cloner will be used regardless of domain access type. The keys are the Gizmo class names (Members cannot be recorded in the Quarkus recorder).